components/core/src/clp/ffi/ir_stream/protocol_constants.hpp

```diff
@@ -12,8 +13,10 @@ constexpr int8_t LengthUByte = 0x11;
 constexpr int8_t LengthUShort = 0x12;
 
 constexpr char VersionKey[] = "VERSION";
-constexpr char VersionValue[] = "0.0.2";
-constexpr char BetaVersionValue[] = "0.1.0-beta.1";
+constexpr std::string_view VersionValue{"0.1.0"};
+constexpr std::string_view MinimumSupportedVersion{"0.1.0-beta.1"};
```

Comment:
Do we need this? Do `0.1.0-beta.1` files exist in the wild, @junhaoliao?

Reply:
Talked with Junhao. I think we can drop support for `0.1.0-beta.1`

Reply:
Should be fine to skip. it's not yet deployed anywhere.